_includes/specials/taproot/en/02-descriptors.md

```diff
@@ -0,0 +1,99 @@
+
+Although descriptors are currently only used by a few projects, they and
+the related [miniscript][topic miniscript] project have the potential to
+significantly improve interoperability between different wallets and
+tools.  This will become increasingly important as more users take
+advantage of the benefits of taproot to improve their security through
+[multisignatures][topic multisignature] and their resilency through
+backup spending conditions.
+
+Before that can happen, descriptors need to be updated to work with
+taproot.  That was the subject of the recently merged [Bitcoin Core #22051][]
+pull request.  The syntax has been designed to allow a single descriptor
+template to provide all the information necessary to use both P2TR
+keypath spends and script path spends.  For simple single-sig, the
+following descriptor is sufficient:
+
+    tr(<key>)
+
+The same syntax may also be used for multisignatures and [threshold
+signatures][topic threshold signature].  For example, Alice, Bob, and
+Carol aggregate their keys using [MuSig][topic musig] and then pay to
+`tr(<combined_key>)`.
+
+Unintuitively, the `key` specified by `tr(<key>)` won't be the key encoded
+in the address.  The `tr()` descriptor follows a [safety recommendation][bip341 safety]
+from BIP341 to use an internal key that commits to an unspendable
+script tree.   This eliminates an attack against users of naive key
+aggregation schemes (more advanced schemes such as [MuSig][topic musig]
+and MuSig2 are not affected).
+
+For scriptpath spending, a new syntax is added that allows specifying
+the contents of a binary tree.  For example, `{ {B,C} , {D,E} }`
+specifies the following tree:
+
+    Internal key
+        / \
+       /   \
+      / \ / \
+      B C D E
+
+A tree can be specified as an optional second parameter to the
+descriptor template we used before.  For example if Alice wants to be
+able to spend via keypath, but she also wants to allow Bob, Carol, Dan,
+and Edmond spend via a scriptpath that generates an audit trail for
+her (but not for third-party chain surveilance), Alice can use the
+following descriptor:
+
+    tr( <a_key> , { {pk(<b_key>),pk(<c_key>)} , {pk(<d_key>),pk(<e_key>)} )
+
+The above features are all that's required to use
+descriptors for taproot, but PR #22051 notes that there are still some
+things missing that could be added to make descriptors better at
+completely describing expected policies:
+
+- **Voided keypaths:** some users may want to prevent usage of keypath
+  spending in order to force scriptpath spending.  That can be done now
+  by using an unspendable key as the first parameter to `tr()`, but it'd
+  be nice to allow wallets to store this preference in the descriptor
+  itself and have it compute a privacy-preserving unspendable keypath.
+
+- **Tapscript multisig:** for legacy and v0 segwit, the `multi()` and
+  `sortedmulti()` descriptors can be used to describe use of the
```

Comment:
Read this through a few times. It's clunky, I think the `used` and `use` being used in close use.